source/sessions/driver-sessions.rst

```diff
@@ -1221,3 +1255,4 @@ Change log
 :2018-06-07: Document that estimatedDocumentCount does not support explicit sessions
 :2018-07-19: Justify why session must be an explicit parameter to each function
 :2018-10-11: Session pools must be cleared in child process after fork
+:2019-05-01: v1.6.0 A ServerSession that is involved in a network error MUST be discarded
```

Comment:
I don't think "v1.6.0" is necessary here. First time I've seen a version included in the change log.

Reply:
Done